listenbrainz/webserver/static/js/src/stats/UserEntity.tsx

```diff
@@ -0,0 +1,422 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
Can we fix this? https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md

Reply:
Bootstrap doesn't support `button` tag in `nav-pills` which leads to formatting issues. Also the drop-down CSS is written with `<a>` in  mind, so changing it to `<button>` is not possible. The issue is not that serious and can be fixed with the upcoming redesign, so I would say we can leave it disabled for now.

Reply:
Should be fixed with [LB-608](https://tickets.metabrainz.org/browse/LB-608)